platform/lib/schema/index.ts

```diff
@@ -107,14 +107,16 @@ class StringSetting extends Setting<string> {
 
     if (this.minLength && value.length < this.minLength) {
       throw new SettingError(
-        `value is [${value}] but it must have a minimum length of [${this.minLength}].`,
+        `value is [${value}] but it must have a minimum length of [${this
+          .minLength}].`,
```

Comment:
It's doing some weird things in template literals :/

(It's basically splitting where it _can_, which is within `${...}`)